support/support.rotation

```diff
@@ -5,41 +5,23 @@
 #@ slack: #serving-questions
 #@ slacklink: https://knative.slack.com/archives/C0186KU7STW
 
-2020-12-14T01:00:00Z | tcnghia
-2020-12-21T01:00:00Z | nak3
-2020-12-28T01:00:00Z | vagababov
-2021-01-04T01:00:00Z | mattmoor
-2021-01-11T01:00:00Z | vagababov
-2021-01-18T01:00:00Z | markusthoemmes
-2021-01-25T01:00:00Z | julz
-2021-02-01T01:00:00Z | ZhiminXiang
-2021-02-08T01:00:00Z | JRBANCEL
-2021-02-15T01:00:00Z | nak3
-2021-02-22T01:00:00Z | yanweiguo
-2021-03-01T01:00:00Z | dprotaso
-2021-03-08T01:00:00Z | tcnghia
-2021-03-15T01:00:00Z | evankanderson
-2021-03-22T01:00:00Z | ZhiminXiang
-2021-03-29T01:00:00Z | yanweiguo
-2021-04-05T01:00:00Z | yanweiguo
-2021-04-12T01:00:00Z | dprotaso
-2021-04-19T01:00:00Z | julz
-2021-04-26T01:00:00Z | nak3
-2021-05-03T01:00:00Z | markusthoemmes
-2021-05-10T01:00:00Z | nak3
-2021-05-17T01:00:00Z | tcnghia
-2021-05-24T01:00:00Z | vagababov
-2021-05-31T01:00:00Z | yanweiguo
-2021-06-07T01:00:00Z | dprotaso
-2021-06-14T01:00:00Z | dprotaso
-2021-06-21T01:00:00Z | julz
-2021-06-28T01:00:00Z | markusthoemmes
-2021-07-05T01:00:00Z | nak3
-2021-07-12T01:00:00Z | yanweiguo
-2021-07-19T01:00:00Z | ZhiminXiang
-2021-07-26T01:00:00Z | dprotaso
-2021-08-02T01:00:00Z | julz
-2021-08-09T01:00:00Z | markusthoemmes
-2021-08-16T01:00:00Z | nak3
-2021-08-23T01:00:00Z | yanweiguo
-2021-08-30T01:00:00Z | ZhiminXiang
+2021-09-27T01:00:00Z | dprotaso
+2021-10-04T01:00:00Z | carlisia
+2021-10-11T01:00:00Z | julz
+2021-10-18T01:00:00Z | markusthoemmes
+2021-10-25T01:00:00Z | nader-ziada
+2021-11-00T01:00:00Z | nak3
+2021-11-01T01:00:00Z | shinigambit
+2021-11-15T01:00:00Z | yanweiguo
+2021-11-22T01:00:00Z | ZhiminXiang
```

Comment:
I will take a long leave from November to December. Could I be put to October?

Reply:
sure